packages/cozy-sharing/src/components/badge.styl

```diff
@@ -8,7 +8,8 @@
     background-clip padding-box
 
     &.--small
-        border .125rem solid var(--white)
+        border .125rem solid 
+        border-color: palette['white']
```

Comment:
In styl, you don't need `:` are they mandatory ? 🤔

Reply:
see the commit message 🙏

Reply:
Sry I just view the change file 😌 thx for the answer

Reply:
I saw that I didn't update the PR description, this is now done with the latest commit message.